### PR TITLE
Setup logrotate for mongos

### PR DESCRIPTION
**Pull request by zeelax**
# What this PR changes:
- Adds logrotate support

### CODE REVIEW

**Comment by antonu17** [comment]
+1

[Approved with PullApprove](https://pullapprove.com/hellofresh/ansible-mongodb-mongos/pull-request/11/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=hellofresh/ansible-mongodb-mongos)

File: defaults/main.yml [hunk 1]
```diff
@@ -11,3 +11,5 @@ mongodb_mongos_listen_host: 127.0.0.1
 mongodb_mongos_listen_port: 27017
 
 mongodb_mognos_start_service: True
+
+mongodb_mongos_log_path: "/var/log/mongodb/mongos.log"
```
Comment (vgarvardt) [inline defaults/main.yml]: newline

**Comment by vgarvardt** [comment]
👍 

[Approved with PullApprove](https://pullapprove.com/hellofresh/ansible-mongodb-mongos/pull-request/11/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=hellofresh/ansible-mongodb-mongos) [Approved with PullApprove](https://pullapprove.com/hellofresh/ansible-mongodb-mongos/pull-request/11/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=hellofresh/ansible-mongodb-mongos)

**Comment by aleerizw-zz** [comment]
+1

[Approved with PullApprove](https://pullapprove.com/hellofresh/ansible-mongodb-mongos/pull-request/11/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=hellofresh/ansible-mongodb-mongos) [Approved with PullApprove](https://pullapprove.com/hellofresh/ansible-mongodb-mongos/pull-request/11/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=hellofresh/ansible-mongodb-mongos) [Approved with PullApprove](https://pullapprove.com/hellofresh/ansible-mongodb-mongos/pull-request/11/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=hellofresh/ansible-mongodb-mongos)

**Pull request merged**